tests/test_schemas/+Decleration/CurleyBracketCommentTable.m

```diff
@@ -0,0 +1,9 @@
+%{
+# subject information
+subject_fullname            : varchar(64)                   # {username}_{subject_nickname}
+---
+subject_nickname            : varchar(16)                   # nickname
+%}
+
+classdef CurleyBracketCommentTable < dj.Manual
```

Comment:
I know it is tempting here to use the first thing that works but... this test example is not the best example of how to actually use DataJoint. A lot of the times our tests become sudo-documentation or an educational means of how others or developers can use DataJoint. We should try to find an example that tests for what we need but also aligns with our design philosophy and methodology i.e. it can serve multiple purposes.

My suggestion would be:
- simply add a curly bracket comment to an existing simple manual  table in the tests, or
- create a schema corresponding to a area of organization, name the table such that it represents the entities stored, and name the attributes with the table name prepended (so they are globally unique).